lms/templates/login.html

```diff
@@ -130,6 +130,8 @@
   </script>
 </%block>
```

Comment:
It doesn't seem to have anything that would care, so I think it is fine where you have it.